### PR TITLE
Uplift third_party/tt_forge_models to origin/main (213a13d57718667429f89f7ec38964598b84bdb2) 2026-05-26

### PR DESCRIPTION
**Pull request by vmilosevic**
This PR uplifts the third_party/tt_forge_models submodule to origin/main

### CODE REVIEW

**Review by nvukobratTT — APPROVED** [review]

**Comment by vmilosevic** [comment]
### List of tenstorrent/tt-forge-models commits since previous uplift:
[213a13d577](https://github.com/tenstorrent/tt-forge-models/commit/213a13d577) Hidream_i1: add shard spec for T5 encoder(text_encoder_3) (#692)
[5af68bbe99](https://github.com/tenstorrent/tt-forge-models/commit/5af68bbe99) Add Mochi-1 full-res input helpers and transformer shard specs (#647)
[320b71a997](https://github.com/tenstorrent/tt-forge-models/commit/320b71a997) Fix DeepSeek-OCR NaN logits on transformers 5.2 after from_pretrained load (#695)
